account_payment_partner/views/report_invoice.xml

```diff
@@ -9,7 +9,7 @@
             </p>
             <p t-if="o.partner_bank_id">
                 <strong>Bank Account:</strong>
-                <t t-esc="o.partner_bank_id.bank_id.name + ('' if not o.partner_bank_id.bank_id.bic else ' (' + o.partner_bank_id.bank_id.bic + ')')" />
+                <t t-esc="(o.partner_bank_id.bank_id.name or '') + ('' if not o.partner_bank_id.bank_id.bic else ' (' + o.partner_bank_id.bank_id.bic + ')')" />
```

Comment:
I think in this case it's better to put directly `<t t-if="o.partner_bank_id.bank_id">`